### PR TITLE
fix: removed space to left in settings

### LINKED ISSUE #55: Align Settings Fragment options
**Actual Behaviour**

Samples per seconds, Bins, Number Channels are not aligned properly

**Expected Behaviour**

Aligned them to left

**Steps to reproduce it**

Open Settings from menu

**Screenshots of the issue**

screenshot_20190218-144618

**Would you like to work on the issue?**

yes

### LINKED ISSUE #73: In settings menu,summary should be shown,and updated value should also be shown
**Actual Behaviour**
the summary are not showing 

Please state here what is currently happening.

**Expected Behaviour**
it show the summary of the editTextPreference
State here what the feature should enable the user to do.
if the user update the summary,the updated value should be shown.

**Screenshots of the issue**
screenshot_20190304-192013
--After changes
screenshot_20190304-192853
--After updating a value
screenshot_20190304-192905

**Would you like to work on the issue?**
I would like to work on this issue
Let us know if this issue should be assigned to you or tell us who you think could help to solve this issue.

### PR DESCRIPTION
**Pull request by CloudyPadmal**
Fixes #55 and Fixes #73 

**Changes**: Removed left side spacing from settings

**Screenshot/s for the changes**: 
<table>
<tr><th>#55</th><th>#73</th></tr>
<tr><td></td><td></td></tr>
</table>

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [X] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [X] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [X] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [X] My code does not contain any extra lines or extra spaces
- [X] I have requested reviews from other members

**APK for testing**: 
[settingsfix.zip](https://github.com/fossasia/neurolab-android/files/3226316/settingsfix.zip)

### CODE REVIEW

File: app/src/main/java/io/neurolab/fragments/NeuroSettingsFragment.java [hunk 1]
```diff
@@ -1,40 +1,58 @@
...
+    public static final String KEY_CHANNELS = "numChannels";
 
-    }
+    private EditTextPreference samplesPref;
+    private EditTextPreference binsPref;
+    private EditTextPreference channelsPref;
+
+    SharedPreferences sharedPreferences;
+
+    public NeuroSettingsFragment() {/**/}
```
Comment (adityastic) [inline app/src/main/java/io/neurolab/fragments/NeuroSettingsFragment.java]: Can be removed

Comment (CloudyPadmal) [reply app/src/main/java/io/neurolab/fragments/NeuroSettingsFragment.java]: Ah missed that. Thanks.. Updated..

**Review by adityastic — APPROVED** [review]

File: app/src/main/java/io/neurolab/fragments/NeuroSettingsFragment.java [hunk 2]
```diff
@@ -1,40 +1,56 @@
...
+    public static final String KEY_SAMPLES = "samples";
+    public static final String KEY_BINS = "bins";
+    public static final String KEY_CHANNELS = "numChannels";
 
-    }
+    private EditTextPreference samplesPref;
+    private EditTextPreference binsPref;
+    private EditTextPreference channelsPref;
+
+    SharedPreferences sharedPreferences;
```
Comment (suresh-gandhi) [inline app/src/main/java/io/neurolab/fragments/NeuroSettingsFragment.java]: We can make this private and use getters for scoping them out if required. (Though I think it won't be required much)

**Review by suresh-gandhi — CHANGES_REQUESTED** [review]

File: app/src/main/java/io/neurolab/fragments/NeuroSettingsFragment.java (hunk 2 above)
Comment (jddeep) [reply app/src/main/java/io/neurolab/fragments/NeuroSettingsFragment.java]: For shared preferences, I don't think we need getters and setters. We can get any pref with required key and I believe there is a method like `getDefaultSharedPreferences` that can be used in any Activity. @suresh-gandhi What do you say? Though I agree we can make it private.

Comment (suresh-gandhi) [reply app/src/main/java/io/neurolab/fragments/NeuroSettingsFragment.java]: @jddeep I agree with you and also meant the same.

**Review by jddeep — APPROVED** [review]
@CloudyPadmal All things seem fine just this https://github.com/fossasia/neurolab-android/pull/254#discussion_r288033340

File: app/src/main/java/io/neurolab/fragments/NeuroSettingsFragment.java (hunk 2 above)
Comment (CloudyPadmal) [reply app/src/main/java/io/neurolab/fragments/NeuroSettingsFragment.java]: Ahh good catch. I reused the stuff there. Will update the PR..

**Pull request merged**